tests/ut_seattlelibv2_repypp_include.py

```diff
@@ -0,0 +1,28 @@
+"""
+ut_seattlelibv2_repypp_include.py -- -- This script tests correct #include statements
+
```

Comment:
That's a very terse description. We should mention the Repy preprocessor (as some other tests do), and what the expected correct outcome is.